content/thesis-topics.md

```diff
@@ -29,3 +29,12 @@ You will usually be supervised by the PhD students and graded + supervised by Pr
 - Analysis of the attack surface of infotainment systems via the broadcast interface [[2]](https://arstechnica.com/cars/2022/02/radio-station-snafu-in-seattle-bricks-some-mazda-infotainment-systems/)
 - Analysis of log files from automotive control units for forensic use
 - Fuzzing of Bluetooth implementations (A2DP, PBAP, SDAP, ...) 
+
+### Digital Forensics Topics 
+
+| Title | Short description | Contact person |
+| ----- | ----------------- | -------------- | 
```

Comment:
why is this new section a table, while the others are lists?
I would prefer either this one to be a list as well or the other ones to be converted to a table.

Reply:
As I indicated in Teams, we should provide additional information. So I would vote for the table-schema.